jwst/pipeline/calwebb_coron3.py

```diff
@@ -85,21 +85,28 @@ def process(self, input):
             psf_models.append(psf_input)
             psf_input.close()
 
+        # Perform outlier detection on the PSFs.
+        if not self.outlier_detection.skip:
+            for model in psf_models:
+                self.outlier_detection(model)
+
         # Stack all the PSF images into a single CubeModel
         psf_stack = self.stack_refs(psf_models)
         psf_models.close()
 
         # Save the resulting PSF stack
         self.save_model(psf_stack, suffix='psfstack')
 
-        # Call the sequence of steps align_refs, klip, and outlier_detection
+        # Call the sequence of steps outlier_detection, align_refs, and klip.
```

Comment:
Minor nit, but should remove the period at the end of this phrase, because the thought is continued on the next comment line.